pilot/pkg/networking/core/v1alpha3/envoyfilter/rc_patch_test.go

```diff
@@ -410,8 +407,7 @@ func TestApplyRouteConfigurationPatches(t *testing.T) {
 				Domains: []string{"gateway", "domain:80"},
 			},
 			{
-				Name:    "new-vhost",
-				Domains: []string{"domain:80"},
```

Comment:
why remove?

Reply:
The "new-vhost" is a new added VirtualHost, we should not allow mutating a user specified patch.